.cicd/platforms/ubuntu22.Dockerfile

```diff
@@ -14,4 +14,7 @@ RUN apt-get update && apt-get upgrade -y && \
                                                       llvm-11-dev          \
                                                       ninja-build          \
                                                       pkg-config           \
+                                                      python3-pip          \
                                                       zstd
+
+RUN python3 -m pip install numpy
```

Comment:
(no need to take action on this comment at the moment -- not until ready to start preparing this for a main merge)

This would seem to add a new requirement to build and test Leap: it needs to be documented in the Readme and such. It's also a bit unfortunate as we've gone 5 years without needing non-standard python packages. I'm not sure if maybe we can detect the presence of this package and simply not clutch in the single test that requires it; doing this opens up other issues though.

Reply:
Updating documentation is in the roadmap downstream a bit.  I'll make sure to note these specific updates to take care of at that point. Thanks.

Regarding `numpy` - While it may seem unfortunate from a certain perspective, it really is the `fundamental package for scientific computing with Python` and very widely adopted and used when doing the types of computing being asked for in a performance monitoring tool.  It is a standard Ubuntu package and extremely well supported, even to the point that there are boost bindings available for it also as a package.

Reply:
That's true, maybe consider going ahead and migrating to the Ubuntu package instead of the pip package. At least for 20 & 22 where it would eliminate the pip stuff altogether.

Reply:
Okay.  I'll add that to our backlog and make sure that's done before this hits `main`.  Thanks.